frontend/src/assets/scss/bt/base/_typography.scss

```diff
@@ -11,80 +11,92 @@ p,
 a,
 button,
 td {
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: normal;
-  color: $bt-base-text;
-  margin: 0;
+	font-family: 'Inter', sans-serif;
+	font-style: normal;
+	font-weight: normal;
+	color: $bt-base-text;
+	margin: 0;
 
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
+
+::-webkit-scrollbar {
```

Comment:
I would actually advise against altering the native appearance of the scrollbar. You will get different results depending on the operating system and browser and you will see unintended side effects unless you account for every edge case. For example, on Chrome on my Macbook, you now force scrollbars to show (when they don't normally) and you did not account for horizontal scrollbars, which have become large, clunky, and visually unappealing.